AdyenCard/Components/Card/CardScannerController.swift

```diff
@@ -62,14 +66,36 @@ internal protocol CardScannerControlling {
                 .failure(CardScannerError.scanningError)
             }
         }
+
+        private func makeNavigationController() -> UINavigationController {
+            guard #available(iOS 13.0, *) else { return UINavigationController() }
+
+            let appearance = UINavigationBarAppearance()
+            appearance.configureWithDefaultBackground()
```

Comment:
A bit rusty on this, will this appearance change not affect all other navigation bars using appearance?

Reply:
`UINavigationBar.appearance()` is for global changes, here I am setting this for a specific navigation bar instance.